src/wp-includes/theme-templates.php

```diff
@@ -154,8 +148,15 @@ function the_block_template_skip_link() {
 			top: 5px;
 			width: auto;
 			z-index: 100000;
-		}
-	</style>
+		}';
+
+	$handle = 'skip-link-styles';
+
+	wp_register_style( $handle, false );
+	wp_add_inline_style( $handle, $skip_link_styles );
+	wp_enqueue_style( $handle );
```

Comment:
No need to include `styles` in the handle is clearly styles already. How about:

```suggestion
	$style_handle = 'wp-block-template-skip-link';

	wp_register_style( $style_handle, false );
	wp_add_inline_style( $style_handle, $skip_link_styles );
	wp_enqueue_style( $style_handle );
```

Also included here is the variable name change anticipating corresponding script change in https://github.com/WordPress/wordpress-develop/pull/4773

Reply:
Really like this change! But changing the variable name doesn't make sense if we use it for styles and scripts. Change the handle.

Reply:
Why can't the handle be the same? They're used in two completely different instances of `WP_Dependencies` and the HTML IDs for scripts and styles get different suffixes, `-js-after` and `-inline-css` respectively.

The same script/style handles are used for:

* `wp-mediaelement`
* `wp-codemirror`
* `site-health`
* `wp-color-picker`
* `thickbox`
* ...